### ISSUE TITLE
BatchPublication

### ISSUE THREAD

**Issue by Axlii50** [allegro/allegro-api#7708]
Dzień dobry mam tworzony obiekt o takiej strukturze
{"offerCriteria":[{"offers":[{"id":"14432313489"},{"id":"14432313563"},{"id":"14432313617"},{"id":"14432313687"},{"id":"14432313755"},{"id":"14432313851"},{"id":"14432313953"},{"id":"14432314236"},{"id":"14432314471"},{"id":"14432314594"},{"id":"14432314712"},{"id":"14432314917"},{"id":"14432315015"},{"id":"14432315113"},{"id":"14432315180"},{"id":"14432315370"},{"id":"14432315667"},{"id":"14432315744"},{"id":"14432315816"},{"id":"14432315877"},{"id":"14432315987"},{"id":"14432316049"},{"id":"14432316139"},{"id":"14432316242"},{"id":"14432316746"},{"id":"14432316814"},{"id":"14432316898"},{"id":""},{"id":"14432317521"},{"id":"14432317686"},{"id":""},{"id":"14432317872"},{"id":"14432317963"},{"id":"14432318035"},{"id":"14432318091"},{"id":"14432318164"},{"id":"14432318229"},{"id":"14432318335"},{"id":"14432318425"},{"id":"14432318517"},{"id":"14432318662"},{"id":""},{"id":""},{"id":""},{"id":"14432318879"},{"id":"14432318932"},{"id":"14432319004"},{"id":"14432319032"},{"id":"14432319083"},{"id":"14432319136"},{"id":"14432319219"},{"id":"14432319381"},{"id":"14432319491"},{"id":"14432319617"},{"id":""},{"id":"14432319835"},{"id":"14432319904"},{"id":"14432320021"},{"id":"14432320151"},{"id":"14432320228"},{"id":"14432320302"},{"id":"14432320401"},{"id":"14432320541"},{"id":"14432320664"},{"id":"14432320790"},{"id":"14432320894"},{"id":"14432320984"},{"id":"14432321078"},{"id":"14432321153"},{"id":"14432321210"},{"id":"14432321285"},{"id":"14432321361"},{"id":"14432321452"},{"id":"14432321524"},{"id":"14432321590"},{"id":"14432321652"},{"id":"14432321731"},{"id":"14432321788"},{"id":"14432321868"},{"id":"14432321950"},{"id":"14432322044"},{"id":"14432322128"},{"id":"14432322201"},{"id":""},{"id":"14432322373"},{"id":"14432322437"},{"id":"14432322666"},{"id":"14432322862"},{"id":"14432322987"},{"id":"14432323069"},{"id":"14432323188"},{"id":"14432323310"},{"id":"14432323407"},{"id":"14432323479"},{"id":"14432323584"},{"id":"14432323652"},{"id":"14432323717"},

lecz mimo ze obiekt jest stworzony tak jak na stronie 
https://developer.allegro.pl/documentation#operation/changePublicationStatusUsingPUT
dalej dostaje komunikat tak jak niżej

{"id":"14432323815"},{"id":"14432323888"},{"id":""}],"type":"CONTAINS_OFFERS"}],"publication":{"action":"ACTIVATE","scheduleFor":null}}
{"errors":[{"code":"VALIDATION_ERROR","message":"nie moze byc puste","details":"Invalid value: ","path":"offerCriteria[0].offers[].id","userMessage":"nie moze byc puste"}]}

**Comment by PrzemyslawLukanowski** [comment]
Zwróc proszę uwagę, że w kilku miejscach definiujesz pole "id", ale przekazujesz dla niego pustą wartość:

<img width="616" alt="Zrzut ekranu 2023-10-3 o 10 07 20" src="https://github.com/allegro/allegro-api/assets/51473661/d2998d4e-36a6-4f56-8db0-5e37f7d503a8">

**Comment by TimothyKoval** [comment]
Poprawcie userMessage

**Comment by PrzemyslawLukanowski** [comment]
Przekazuję informację do zespołu odpowiedzialnego za zasób.

**Comment by Axlii50** [comment]
i rozumiem ze prze te puste pola wywala błąd?

**Comment by PrzemyslawLukanowski** [comment]
Tak, w polu `path` z odpowiedzi błędu zwracamy ścieżkę `"offerCriteria[0].offers[].id""`, a w polu `userMessage` i `message` przyczynę błędu - `"nie moze byc puste".`
